az-digital/az_quickstart
# #3017 Deprecate the Block Content Permissions module and no longer install by default

## Description

Deprecates the [Block Content Permissions module](https://www.drupal.org/project/block_content_permissions) and changes Quickstart to no longer install it by default since the module is no longer needed as of Drupal 10.1 (Quickstart 2.8).

## Related issues




#3017 

## How to test


Link to Probo review site: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-3020.probo.build

1. Confirm that the Block Content Permissions module was not installed by default.
2. After installing the module, confirm that the Status Report page shows a "Quickstart deprecated modules enabled" warning for the module.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
